apps/web/src/components/commands/tokens/Config.tsx

```diff
@@ -0,0 +1,120 @@
+import { NodeViewWrapper } from "@tiptap/react";
+import React, { useEffect, useMemo, useState } from "react";
+
+import * as Popover from "@radix-ui/react-popover";
+import * as chains from "wagmi/chains";
```

Comment:
👀

I guess we could have used wagmi's chains and filtered based on the `network` property:
```ts
export declare type Chain = {
    /** ID in number form */
    id: number;
    /** Human-readable name */
    name: string;
    /** Internal network name */
    network: string; // i.e. 'arbitrum'
    /** Currency used by chain */
    nativeCurrency?: AddEthereumChainParameter['nativeCurrency'];
    /** Collection of RPC endpoints */
    rpcUrls: {
        [key in RpcProviderName]?: string;
    } & {
        [key: string]: string;
        default: string;
    };
    /** Collection of block explorers */
    blockExplorers?: {
        [key in BlockExplorerName]?: BlockExplorer;
    } & {
        [key: string]: BlockExplorer;
        default: BlockExplorer;
    };
    /** ENS registry */
    ens?: {
        address: string;
    };
    /** Chain multicall contract */
    multicall?: {
        address: string;
        blockCreated: number;
    };
    /** Flag for test networks */
    testnet?: boolean;
};
```

Reply:
Hm, I don't understand this comment. Instead of the `supportedChains`. The reason was that it seemed to differ from which networks Portals.fi supported